### PR TITLE
Update resources-list.md

### PR DESCRIPTION
**Pull request by GauranshiGupta**

### CODE REVIEW

**Comment by alodahl** [comment]
Which issue does this address?

File: women-in-tech-resources/resources-list.md [hunk 1]
```diff
@@ -164,7 +164,8 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 - [Lets Intern](https://www.letsintern.com/)
 - [Intern Supply](https://intern.supply/)
 - [Techgig](https://www.techgig.com/)
-
+- [Twenty19](http://www.twenty19.com/)
+- [Hello Intern](https://www.hellointern.com/)
```
Comment (alodahl) [inline women-in-tech-resources/resources-list.md]: Twenty19 looks good, but Hello Intern doesn’t seem to have many resources. Can you take it out?

Comment (GauranshiGupta) [reply women-in-tech-resources/resources-list.md]: Resolved!

**Comment by alodahl** [comment]
Thanks @GauranshiGupta ! Please consider taking another easy issue from out To-do list [here](https://github.com/Techtonica/curriculum/projects)!

**Pull request merged**